### PR TITLE
XWIKI-15641: Display information to user when a job is waiting another to finish/timeout

### PR DESCRIPTION
**Pull request by surli**
## Issue

https://jira.xwiki.org/browse/XWIKI-15641

This PR is complementary to: https://github.com/xwiki/xwiki-platform/pull/1051
It aims at fixing a bug I introduced which force the user to reload the page when the blocking job is finished. 

## Solution

  * Process the info message in JS to remove it when the blocking job is
finished.

### CODE REVIEW

**Comment by surli** [comment]
@mflorea could you review this one please?

File: xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/job/job.js [hunk 1]
```diff
@@ -33,6 +33,15 @@ require.config({
 require(['jquery', 'xwiki-meta', 'JobRunner'], function($, xm, JobRunner) {
   'use strict';
   var updateProgress = function(jobUI, job) {
+    if (job.state === 'NONE') {
+      jobUI.find('.ui-progress-background').css('display', 'none');
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/job/job.js]: See https://api.jquery.com/hide/#hide

> This is roughly equivalent to calling .css( "display", "none" )

File: xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/job/job.js [hunk 2]
```diff
@@ -33,6 +33,15 @@ require.config({
...
+      jobUI.find('.ui-progress-message').css('display', 'none');
+    } else {
+      jobUI.find('#state-none-hint').remove();
+      jobUI.find('.ui-progress-background').css('display', 'block');
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/job/job.js]: See https://api.jquery.com/show/#show

> This is roughly equivalent to calling .css( "display", "block" )

File: xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/job/job.js [hunk 3]
```diff
@@ -33,6 +33,15 @@ require.config({
...
+      jobUI.find('.ui-progress-message').css('display', 'block');
+    }
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/job/job.js]: Another option is to use:

    jobUI.find('.ui-progress-background').toggle(job.state !== 'NONE')

See https://api.jquery.com/toggle/#toggle-display .

Comment (surli) [reply xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/job/job.js]: Hm indeed it looks appropriate here.

**Comment by surli** [comment]
ping @mflorea this one is now ready

**Pull request merged**